libnavigation-ui/src/main/java/com/mapbox/navigation/ui/feedback/FeedbackAdapter.java

```diff
@@ -0,0 +1,57 @@
+package com.mapbox.navigation.ui.feedback;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.mapbox.libnavigation.ui.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import androidx.recyclerview.widget.RecyclerView;
+
+public class FeedbackAdapter extends RecyclerView.Adapter<FeedbackViewHolder> {
+
+  private static final String EMPTY_FEEDBACK_DESCRIPTION = "";
+  private List<FeedbackItem> feedbackItems = new ArrayList<>();
+
+  FeedbackAdapter(Context context) {
+    // TODO Telemetry impl
```

Comment:
Let's cut tickets with this `TODO` and the rest 👇 so we don't forget 👍

Reply:
https://github.com/mapbox/mapbox-navigation-android/issues/2507